package.json

```diff
@@ -29,38 +29,38 @@
   },
   "dependencies": {
     "@jenkins-cd/eslint-config-jenkins": "0.0.2",
-    "babel-preset-es2015": "^6.5.0",
-    "babelify": "^7.x",
-    "brfs": "^1.4.3",
-    "browser-pack": "^6.0.1",
-    "browser-unpack": "^1.1.1",
-    "browserify": "^12.0.1",
-    "browserify-transform-tools": "^1.4.2",
+    "babel-preset-es2015": "6.16.0",
```

Comment:
:bug: removing the caret breaks things from building in BO, can't do that until we figure out how to fix

Reply:
... actually may have spoken too soon, did you also update the deps such that BO builds? this might be fine, then

Reply:
Hmm, can you be specific? I see https://github.com/jenkinsci/blueocean-plugin/pull/1508 had no problem and it is using the pre-published version. That package.json **should never had** use carets to begin with.

Reply:
@scherler agree, it should never have used carets, however when tom removed carets in master, it broke building in BO. But you're right, I think your changes seem to work here. Could you make a separate PR for this change, please?

Reply:
I would but because of https://github.com/jenkinsci/js-builder/pull/22/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R49 vs. https://github.com/jenkinsci/js-builder/pull/22/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L49 that makes no sense.